src/api/app/policies/comment_policy.rb

```diff
@@ -66,4 +66,10 @@ def locked?
       record.commentable.comment_lock.present?
     end
   end
+
+  def history?
+    return false unless Flipper.enabled?(:content_moderation, user)
+
+    !(record.moderated? || record.user.is_nobody?) || (user.is_admin? || user.is_staff? || user.is_moderator?)
```

Comment:
I find this a bit difficult to understand. Maybe rework the booleans or add a comment...

Reply:
Better